test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -35,6 +35,24 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
     }
+    
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
No need for the `throws Exception` since `Utils.isAnyNull` does not throw checked exceptions.

Reply:
Thanks for the catch. Removed.